### DISCUSSION TITLE
May 29 Sync

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#179] [Q&A] [answered]
@hollowaykeanho , update?

**Comment by hollowaykeanho** [comment, marked as answer]
`/en/releases` pages are pending porting (start today)
`/en/specs` pages are pending porting

Both should be doable this week since they're very data centrie (same layout design, different dataset).

----

Although we solved the iterative performance problem, we have a "memory leak" problem from Hugo. Still optimizing along the way to see we can deter it. https://github.com/ZORALab/Hestia/issues/106

Efforts of transitioning to NIM is on-going. Someone from Taiwan requested to keep `hestiaRS` rather than deleting it in case of soft-landing when u-turning. https://github.com/ZORALab/Hestia/discussions/175

**Comment by corygalyna** [comment]
Does that mean we can anticipate a `1.2.0` release somewhere next week?

**Comment by hollowaykeanho** [comment]
Yes.

**Comment by corygalyna** [comment]
That's great news! Proceed as planned.
